app.ts

```diff
@@ -490,7 +494,7 @@ async function main() {
 
     if (opts.prediscovered) {
         const prediscoveredCompilersJson = await fs.readFile(opts.prediscovered);
-        initialCompilers = JSON.parse(prediscoveredCompilersJson);
+        initialCompilers = JSON.parse(prediscoveredCompilersJson.toString());
         await compilerFinder.loadPrediscovered(initialCompilers);
```

Comment:
fs-extra delegates readFile to fs, so we can use tell it to read as "utf-8", removing the need to .toString()

https://nodejs.org/api/fs.html#filehandlereadfileoptions

Reply:
Ah perfect thanks